api/v1alpha1/glideinmanagerpilotset_types.go

```diff
@@ -33,8 +34,33 @@ type GlideinManagerPilotSetSpec struct {
 	// rather than the clients
 	GlideinManagerUrl string `json:"glideinManagerUrl,omitempty"`
 
+	// Collection of Glidein deployment specs
+	GlideinSets []GlideinSetSpec `json:"glideinSets"`
+}
+
+// Collection of Glideins sharing a priority class, per-pod resource allocation, and node affinity
+type GlideinSetSpec struct {
```

Comment:
I was under the impression that lowercased variables were file-private rather than package-private, which appears to be incorrect. I think given that it's correct for basically everything to be lowercased/private here